specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-06-01-preview/examples/confidentialWatchlists/GetConfidentialWatchlistItems.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "api-version": "2022-06-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "operationalInsightsResourceProvider": "Microsoft.OperationalInsights",
+    "watchlistAlias": "highValueAsset"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/confidentialWatchlists/highValueAsset/WatchlistItems/fd37d325-7090-47fe-851a-5b5a00c3f576",
+            "name": "fd37d325-7090-47fe-851a-5b5a00c3f576",
+            "etag": "\"f2089bfa-0000-0d00-0000-601c58b42021\"",
+            "type": "Microsoft.SecurityInsights/confidentialWatchlists/WatchlistItems",
+            "properties": {
+              "watchlistItemType": "watchlist-item",
+              "watchlistItemId": "fd37d325-7090-47fe-851a-5b5a00c3f576",
+              "tenantId": "3f8901fe-63d9-4875-9ad5-9fb3b8105797",
+              "isDeleted": false,
+              "created": "2021-02-04T12:27:32.3783333-08:00",
+              "updated": "2021-02-04T12:27:32.3783333-08:00",
+              "createdBy": {
+                "objectId": "2046feea-040d-4a46-9e2b-91c2941bfa70",
+                "email": "john@contoso.com",
+                "name": "john doe"
+              },
+              "updatedBy": {
+                "objectId": "2046feea-040d-4a46-9e2b-91c2941bfa70",
+                "email": "john@contoso.com",
+                "name": "john doe"
+              },
```

Comment:
Consider using systemData to capture these metadata.
[https://github.com/Azure/azure-rest-api-specs/blob/49135492d5ea1c15f1ba53813c4be8cb2109c416/specification/common-types/resource-management/v3/types.json#L528](https://github.com/Azure/azure-rest-api-specs/blob/49135492d5ea1c15f1ba53813c4be8cb2109c416/specification/common-types/resource-management/v3/types.json#L528)